api/sign.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+var sjcl = require('ripple-lib').sjcl;
+var Seed = require('ripple-lib').Seed;
+var SerializedObject = require('ripple-lib').SerializedObject;
+var validate = require('./lib/validate');
+
+var HASH_TX_ID = 0x54584E00; // 'TXN'
```

Comment:
That's in a different project and not currently exposed. There's not much benefit to exposing it considering it can never change. But we can change it if/when this moves to the same project.